frontend/src/components/header/Header.vue

```diff
@@ -43,18 +40,18 @@ const props = defineProps({
             </Button>
             <span class="header-title">
                 {{ props.title }}
-                <strong class="subtitle">{{ props.subtitle }} {{ environmentLabel }}</strong>
+                <strong class="subtitle"
+                    >{{ props.subtitle }} {{ environmentLabel }}</strong
+                >
```

Comment:
Wondering if it is the VS code breaking the tag like this with strange looking "**</strong**" and 
"**>**" at the next line?

Reply:
I think this is done by auto formatting. I tried again, it auto formats this way.